uaclient/clouds/tests/test_azure.py

```diff
@@ -124,13 +124,30 @@ def fake_load_file(f_name):
         instance = UAAutoAttachAzureInstance()
         assert viable is instance.is_viable
 
-    def test_unsupported_should_poll_for_pro_license(self):
-        """Unsupported"""
+    def test_should_poll_for_license(self):
         instance = UAAutoAttachAzureInstance()
-        assert not instance.should_poll_for_pro_license()
+        result = instance.should_poll_for_pro_license()
+        assert result
 
-    def test_unsupported_is_pro_license_present(self):
-        """Unsupported"""
+    @pytest.mark.parametrize(
+        "metadata_response, expected_result",
+        (
+            (({"licenseType": ""}, {}), False),
```

Comment:
one more little test case here, where there is no `licenseType` in the response? should never happen but Christian mentioned it would be nice to cover